theano/tensor/nnet/abstract_conv.py

```diff
@@ -177,6 +177,187 @@ def conv2d(input,
     return conv_op(input, filters)
 
 
+def deconv(input,
+           filters,
+           input_shape=None,
+           output_shape=None,
+           filter_shape=None,
+           border_mode='valid',
+           subsample=(1, 1),
+           filter_flip=True):
+    """This function will build the symbolic graph for deconvoloving a
+    mini-batch of a stack of 2D inputs with a set of 2D filters (using
+    gradient of convolution), such that it upsamples the input to the
+    desired output resolution.
```

Comment:
I definitely agree!! In ReSeg we called it *Gradient-based upscaling layer*, which is not very informative but more accurate.